app/components/account/OrderHistoryItem.tsx

```diff
@@ -0,0 +1,171 @@
+                                <span className="flex flex-1 flex-col gap-0">
+                                    {/* Product name */}
+                                    <Link
+                                        to={`/products/${line.productVariant.product.slug}`}
+                                        className="text-black text-sm font-semibold line-clamp-3 md:line-clamp-2 max-w-md hover:text-black/50"
+                                        title={line.productVariant.name}
+                                    >
+                                        {line.productVariant.name}
+                                    </Link>
+                                    {/* Price and quantity */}
+                                    <button className="inline-flex gap-2 items-center w-fit text-gray-500 text-sm mt-1" onClick={() => setIsLineCalcExpanded(!isLineCalcExpanded)}>
+                                        {isLineCalcExpanded && (<>
+                                        <span title="Quantity">{line.quantity}</span>
+                                        <span className="text-gray-300 select-none">×</span>
+                                        <span title="Price per unit"><Price currencyCode={line.productVariant.currencyCode} priceWithTax={line.discountedUnitPriceWithTax}></Price></span>
+                                        <span className="text-gray-300 select-none">Ξ</span>
+                                        </>)}
+                                        <span title="Subtotal"><Price currencyCode={line.productVariant.currencyCode} priceWithTax={line.discountedLinePriceWithTax}></Price></span>
+                                    </button>
+                                    {/* Shipment status */}
+                                    <span className="text-gray-500 text-xs mt-2 tracking-wide">
+                                        {line.fulfillments && line.fulfillments.length === 0 && ('Not shipped yet')}
+                                        {line.fulfillments?.map((f, i) => (
+                                            <span key={i} className="block" title={(new Date(f.updatedAt)).toLocaleString()}>
+                                                {f.state}: {new Intl.DateTimeFormat(undefined, { dateStyle: "medium" }).format(new Date(f.updatedAt))}
+                                            </span>
+                                        ))}
+                                    </span>
+                                </span>
+                            </div>
+                        </div>
+                    ))}
+
+                    {/* Per order actions */}
+                    <div className="p-2 lg:py-3 lg:px-6 flex justify-end gap-2 lg:gap-6">
+                        {/* <Button disabled title="Not implemented">
```

Comment:
I don't mind if we show these as disabled buttons. Since you kept the bar (which isn't a bad thing) this would show that it can be used for other things that are up to customization. My previous "instructions" weren't really clear which I apologise for.

Reply:
No worries, I understood you, I removed them because I dont like it personally. Makes the page feel too cluttered imo, especially on mobile. I included the placeholder three-dot button right next to the expand-chevron top right as a place for future per-order-actions. Someone could easily expand that with a dropdown that shows invoice, support, whatever. :+1: 

I can see the bottom actionbar used for temporary actions you might want more visible like shipment-tracking-link or a message of some kind.